packages/core/src/client/index.ts

```diff
@@ -6,11 +7,8 @@ import * as schema from '../schema'
 const { Pool } = pg
 
 export type Database = NodePgDatabase<typeof schema>
-export function buildDatabaseClient({
-  connectionString,
-}: {
-  connectionString: string
-}): Database {
-  const pool = new Pool({ connectionString })
-  return drizzle(pool, { schema })
-}
+
+const pool = new Pool({ connectionString: env.DATABASE_URL })
```

Comment:
Wait? So env db variable is handled by core?

If this works we can simplify `Transaction` cc @csansoon

Reply:
yes, there's only one db connection declared in core, don't wanna see any postgres references in web or API